### PR TITLE
trivial fixes

### PR DESCRIPTION
**Pull request by luzpaz**
@wmayer I didn't want to spam everyone with this PR. I was wondering  if the whitespace/formatting commit is appropriate as well ?

Edit: I did realize that your master is behind. So the relevant commits I've submitted are  
9508e62  
5ad4315

### CODE REVIEW

**Comment by wwmayer** [comment]
I don't understand why I should merge PR to my clone of the official FreeCAD repo.

**Comment by luzpaz** [comment]
Didn't mean to inconvenience. I looked at the relatively most up to date branch and your master was one of them. Other folks just merge it in to whatever branch they see fit. If this is a imposition then tell me which branch to resubmit to or I'll just close the PR all together.

**Comment by wwmayer** [comment]
If I am not totally wrong then your PR includes all the changes that are already added to official FreeCAD master.
From time to time I sync my clone with the official repo to make sure it has an identical history. Don't know what would happen if I merged PR to my clone, and especially I don't know if this can cause merge conflicts with PR I sometimes open to be merged into the official master branch.

So long story short: I won't add any PRs to my clone as otherwise I had to make sure the history doesn't diverge from the official master.

**Comment by luzpaz** [comment]
I don't think you should merge this PR to your clone. But instead my intention is that you commit:
9508e62
5ad4315
to FreeCAD master without me having to have submit a PR to FreeCAD master. Makes sense? 😆 
But I have caused enough of a ruckus already. So I'll just submit a PR to FC master. Thank you for your time and apologies.

**Comment by wwmayer** [comment]
Just included your two changes now. 
> I was wondering if the whitespace/formatting commit is appropriate as well ?

Basically I leave a certain freedom to the devs' coding style so that we should not start to reformat all code. What however could be changed is removing trailing tabs or spaces but there might be some code beautifiers around that could do it automatically.
